packages/react/src/errorboundary.tsx

```diff
@@ -138,7 +138,14 @@ class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoundarySta
     if (error) {
       let element: React.ReactElement | undefined = undefined;
       if (typeof fallback === 'function') {
-        element = fallback({ error, componentStack, resetError: this.resetErrorBoundary, eventId });
+        element = fallback({
+          error,
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          componentStack: componentStack!,
```

Comment:
Just to be clear (absolutely no expert here), we are 100% positive that when `error` is set in the state, `componentStack` and `eventId` are also never null?

Reply:
Wait, nevermind, just read the related issue, seems good!